tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, pypy, py34, py35, py36, py37, pypy3
+envlist = py{27,py,34,35,36,37,38,py3}
```

Comment:
IMHO the previous format is more readable.

Reply:
Thanks. Perhaps this boils down to personal preference, but the style here is _extremely_ common in other `tox.ini` files. So should be familiar to those that have worked with tox before. It reduces the repetition of the `py` prefix and reduces the amount of change needed with adding or removing a Python version.

If appdirs maintainers prefer the old form, I'll happily comply, just let me know.